R/table.R

```diff
@@ -63,31 +63,49 @@ html_table.xml_node <- function(x, header = NA, trim = TRUE,
 
   ncols <- lapply(cells, html_attr, "colspan", default = "1")
   ncols <- lapply(ncols, as.integer)
+  nrows <- lapply(cells, html_attr, "rowspan", default = "1")
+  nrows <- lapply(nrows, as.integer)
```

Comment:
I wonder if this should be `vapply(nrows, as.integer, integer(1)` so we get an integer vector instead of a list. That would make the rest of the code simpler